### PR TITLE
Added Manual Installation guide

### LINKED ISSUE #55: Manual Installation
`CocoaPods` and `Carthage` are awesome tools and make our life really easier, but there are some devs who still don't know how to use them. 

It would be cool to add the _Manual installation_ guide in your `README.md`. You can take a look at my [iOS Readme Template](https://github.com/awesome-labs/iOS-readme-template) to see how you can do it.

### PR DESCRIPTION
**Pull request by lfarah**
fixes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,10 @@ post_install do |installer|
 end
 ```
 
+#### Manually
+1. Download and drop ```/SwiftTweaks``` folder in your project.
```
Comment (NachoSoto) [inline README.md]: Hmm I don't think that's enough, right?

We haven't attached a precompiled framework to the release, so by `download`, do you mean the whole repo?

Comment (NachoSoto) [inline README.md]: Hmm I don't think that's enough, right?

We haven't attached a precompiled framework to the release, so by `download`, do you mean the whole repo?

Comment (lfarah) [inline README.md]: yeah, that's what most libraries do when they didn't build the framework yet

Comment (bryanjclark) [inline README.md]: These installation instructions aren't sufficient; there's a dependency on a xcconfigs submodule. See #61 for the submodule installation instructions that should be included as well.

Thanks!

**Comment by bryanjclark** [comment]
Hey, I'm going to close this since it's getting stale. Thanks!

**Pull request closed without merging**